src/components/NFTBook/ItemCard.vue

```diff
@@ -424,7 +434,13 @@ export default {
   },
   methods: {
     async fetchInfo() {
-      await this.lazyFetchNFTClassAggregatedData();
+      this.isLoading = true;
+      try {
+        await this.lazyFetchNFTClassAggregatedData();
+      } catch (error) {
```

Comment:
no need to `catch` here if we only need `finally`?

Reply:
[🔥 Remove unused catch](https://github.com/likecoin/liker-land/pull/1896/commits/42239182ecf3004272857a90f8b46ee9ff6286b8)